caiman/source_extraction/cnmf/estimates.py

```diff
@@ -1,25 +1,28 @@
 #!/usr/bin/env python
 
 import bokeh
+import cv2
 import logging
 import matplotlib.pyplot as plt
 import numpy as np
+import os
+from pynwb import NWBHDF5IO, TimeSeries, NWBFile
+from pynwb.base import Images
+from pynwb.image import GrayscaleImage
+from pynwb.ophys import ImageSegmentation, Fluorescence, OpticalChannel, ImageSeries
+from pynwb.device import Device
 import scipy.sparse
 import time
+import uuid
 
 import caiman
-from .utilities import detrend_df_f, decimation_matrix
-from .spatial import threshold_components
-from .temporal import constrained_foopsi_parallel
-from .merging import merge_iteration, merge_components
-from ...components_evaluation import (
-        evaluate_components_CNN, estimate_components_quality_auto,
-        select_components_from_metrics, compute_eccentricity)
-from ...base.rois import (
-        detect_duplicates_and_subsets, nf_match_neurons_in_binary_masks,
-        nf_masks_to_neurof_dict)
-from .initialization import downscale
-
+import caiman.base.rois
```

Comment:
I think I am coming to a one dot limit in what I want to read later in the code.